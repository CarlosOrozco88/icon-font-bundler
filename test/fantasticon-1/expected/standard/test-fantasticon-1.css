@font-face {
    font-family: "test-fantasticon-1";
    src: url("./test-fantasticon-1.ttf?2222f7b806a58c7905dbf04d4db888e2") format("truetype"),
url("./test-fantasticon-1.eot?2222f7b806a58c7905dbf04d4db888e2#iefix") format("embedded-opentype"),
url("./test-fantasticon-1.woff2?2222f7b806a58c7905dbf04d4db888e2") format("woff2"),
url("./test-fantasticon-1.woff?2222f7b806a58c7905dbf04d4db888e2") format("woff");
}

i[class^="icon-"]:before, i[class*=" icon-"]:before {
    font-family: test-fantasticon-1 !important;
    font-style: normal;
    font-weight: normal !important;
    font-variant: normal;
    text-transform: none;
    line-height: 1;
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
}

.icon-arrow-90deg-down:before {
    content: "\f101";
}
.icon-arrow-90deg-left:before {
    content: "\f102";
}
.icon-arrow-90deg-right:before {
    content: "\f103";
}
.icon-arrow-90deg-up:before {
    content: "\f104";
}
.icon-arrow-bar-down:before {
    content: "\f105";
}
.icon-arrow-bar-left:before {
    content: "\f106";
}
.icon-arrow-bar-right:before {
    content: "\f107";
}
.icon-arrow-bar-up:before {
    content: "\f108";
}
.icon-arrow-clockwise:before {
    content: "\f109";
}
.icon-arrow-counterclockwise:before {
    content: "\f10a";
}
.icon-arrow-down-circle-fill:before {
    content: "\f10b";
}
.icon-arrow-down-circle:before {
    content: "\f10c";
}
.icon-arrow-down-left-circle-fill:before {
    content: "\f10d";
}
.icon-arrow-down-left-circle:before {
    content: "\f10e";
}
.icon-arrow-down-left-square-fill:before {
    content: "\f10f";
}
.icon-arrow-down-left-square:before {
    content: "\f110";
}
.icon-arrow-down-left:before {
    content: "\f111";
}
.icon-arrow-down-right-circle-fill:before {
    content: "\f112";
}
.icon-arrow-down-right-circle:before {
    content: "\f113";
}
.icon-arrow-down-right-square-fill:before {
    content: "\f114";
}
.icon-arrow-down-right-square:before {
    content: "\f115";
}
.icon-arrow-down-right:before {
    content: "\f116";
}
.icon-arrow-down-short:before {
    content: "\f117";
}
.icon-arrow-down-square-fill:before {
    content: "\f118";
}
.icon-arrow-down-square:before {
    content: "\f119";
}
.icon-arrow-down-up:before {
    content: "\f11a";
}
.icon-arrow-down:before {
    content: "\f11b";
}
.icon-arrow-left-circle-fill:before {
    content: "\f11c";
}
.icon-arrow-left-circle:before {
    content: "\f11d";
}
.icon-arrow-left-right:before {
    content: "\f11e";
}
.icon-arrow-left-short:before {
    content: "\f11f";
}
.icon-arrow-left-square-fill:before {
    content: "\f120";
}
.icon-arrow-left-square:before {
    content: "\f121";
}
.icon-arrow-left:before {
    content: "\f122";
}
.icon-arrow-repeat:before {
    content: "\f123";
}
.icon-arrow-return-left:before {
    content: "\f124";
}
.icon-arrow-return-right:before {
    content: "\f125";
}
.icon-arrow-right-circle-fill:before {
    content: "\f126";
}
.icon-arrow-right-circle:before {
    content: "\f127";
}
.icon-arrow-right-short:before {
    content: "\f128";
}
.icon-arrow-right-square-fill:before {
    content: "\f129";
}
.icon-arrow-right-square:before {
    content: "\f12a";
}
.icon-arrow-right:before {
    content: "\f12b";
}
.icon-arrow-up-circle-fill:before {
    content: "\f12c";
}
.icon-arrow-up-circle:before {
    content: "\f12d";
}
.icon-arrow-up-left-circle-fill:before {
    content: "\f12e";
}
.icon-arrow-up-left-circle:before {
    content: "\f12f";
}
.icon-arrow-up-left-square-fill:before {
    content: "\f130";
}
.icon-arrow-up-left-square:before {
    content: "\f131";
}
.icon-arrow-up-left:before {
    content: "\f132";
}
.icon-arrow-up-right-circle-fill:before {
    content: "\f133";
}
.icon-arrow-up-right-circle:before {
    content: "\f134";
}
.icon-arrow-up-right-square-fill:before {
    content: "\f135";
}
.icon-arrow-up-right-square:before {
    content: "\f136";
}
.icon-arrow-up-right:before {
    content: "\f137";
}
.icon-arrow-up-short:before {
    content: "\f138";
}
.icon-arrow-up-square-fill:before {
    content: "\f139";
}
.icon-arrow-up-square:before {
    content: "\f13a";
}
.icon-arrow-up:before {
    content: "\f13b";
}
.icon-arrows-angle-contract:before {
    content: "\f13c";
}
.icon-arrows-angle-expand:before {
    content: "\f13d";
}
.icon-arrows-collapse:before {
    content: "\f13e";
}
.icon-arrows-expand:before {
    content: "\f13f";
}
.icon-arrows-fullscreen:before {
    content: "\f140";
}
.icon-arrows-move:before {
    content: "\f141";
}
.icon-file-alt:before {
    content: "\f142";
}
.icon-file-archive:before {
    content: "\f143";
}
.icon-file-audio:before {
    content: "\f144";
}
.icon-file-code:before {
    content: "\f145";
}
.icon-file-contract:before {
    content: "\f146";
}
.icon-file-csv:before {
    content: "\f147";
}
.icon-file-download:before {
    content: "\f148";
}
.icon-file-excel:before {
    content: "\f149";
}
.icon-file-export:before {
    content: "\f14a";
}
.icon-file-image:before {
    content: "\f14b";
}
.icon-file-import:before {
    content: "\f14c";
}
.icon-file-invoice-dollar:before {
    content: "\f14d";
}
.icon-file-invoice:before {
    content: "\f14e";
}
.icon-file-medical-alt:before {
    content: "\f14f";
}
.icon-file-medical:before {
    content: "\f150";
}
.icon-file-pdf:before {
    content: "\f151";
}
.icon-file-powerpoint:before {
    content: "\f152";
}
.icon-file-prescription:before {
    content: "\f153";
}
.icon-file-signature:before {
    content: "\f154";
}
.icon-file-upload:before {
    content: "\f155";
}
.icon-file-video:before {
    content: "\f156";
}
.icon-file-word:before {
    content: "\f157";
}
.icon-file:before {
    content: "\f158";
}
