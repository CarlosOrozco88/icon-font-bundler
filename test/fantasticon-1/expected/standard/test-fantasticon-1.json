{
  "arrow-90deg-down": "f101",
  "arrow-90deg-left": "f102",
  "arrow-90deg-right": "f103",
  "arrow-90deg-up": "f104",
  "arrow-bar-down": "f105",
  "arrow-bar-left": "f106",
  "arrow-bar-right": "f107",
  "arrow-bar-up": "f108",
  "arrow-clockwise": "f109",
  "arrow-counterclockwise": "f10a",
  "arrow-down-circle-fill": "f10b",
  "arrow-down-circle": "f10c",
  "arrow-down-left-circle-fill": "f10d",
  "arrow-down-left-circle": "f10e",
  "arrow-down-left-square-fill": "f10f",
  "arrow-down-left-square": "f110",
  "arrow-down-left": "f111",
  "arrow-down-right-circle-fill": "f112",
  "arrow-down-right-circle": "f113",
  "arrow-down-right-square-fill": "f114",
  "arrow-down-right-square": "f115",
  "arrow-down-right": "f116",
  "arrow-down-short": "f117",
  "arrow-down-square-fill": "f118",
  "arrow-down-square": "f119",
  "arrow-down-up": "f11a",
  "arrow-down": "f11b",
  "arrow-left-circle-fill": "f11c",
  "arrow-left-circle": "f11d",
  "arrow-left-right": "f11e",
  "arrow-left-short": "f11f",
  "arrow-left-square-fill": "f120",
  "arrow-left-square": "f121",
  "arrow-left": "f122",
  "arrow-repeat": "f123",
  "arrow-return-left": "f124",
  "arrow-return-right": "f125",
  "arrow-right-circle-fill": "f126",
  "arrow-right-circle": "f127",
  "arrow-right-short": "f128",
  "arrow-right-square-fill": "f129",
  "arrow-right-square": "f12a",
  "arrow-right": "f12b",
  "arrow-up-circle-fill": "f12c",
  "arrow-up-circle": "f12d",
  "arrow-up-left-circle-fill": "f12e",
  "arrow-up-left-circle": "f12f",
  "arrow-up-left-square-fill": "f130",
  "arrow-up-left-square": "f131",
  "arrow-up-left": "f132",
  "arrow-up-right-circle-fill": "f133",
  "arrow-up-right-circle": "f134",
  "arrow-up-right-square-fill": "f135",
  "arrow-up-right-square": "f136",
  "arrow-up-right": "f137",
  "arrow-up-short": "f138",
  "arrow-up-square-fill": "f139",
  "arrow-up-square": "f13a",
  "arrow-up": "f13b",
  "arrows-angle-contract": "f13c",
  "arrows-angle-expand": "f13d",
  "arrows-collapse": "f13e",
  "arrows-expand": "f13f",
  "arrows-fullscreen": "f140",
  "arrows-move": "f141",
  "file-alt": "f142",
  "file-archive": "f143",
  "file-audio": "f144",
  "file-code": "f145",
  "file-contract": "f146",
  "file-csv": "f147",
  "file-download": "f148",
  "file-excel": "f149",
  "file-export": "f14a",
  "file-image": "f14b",
  "file-import": "f14c",
  "file-invoice-dollar": "f14d",
  "file-invoice": "f14e",
  "file-medical-alt": "f14f",
  "file-medical": "f150",
  "file-pdf": "f151",
  "file-powerpoint": "f152",
  "file-prescription": "f153",
  "file-signature": "f154",
  "file-upload": "f155",
  "file-video": "f156",
  "file-word": "f157",
  "file": "f158"
}